File: dotnetcore-k8s-demo/dotnetcore-k8s-demo/Controllers/HostInfo.cs
namespace dotnetcore_k8s_demo.Controllers
{
    public class HostInfo
    {
        public string Ip { get; set; }
        public string Hostname { get; set; }
        public string Version { get; set; }

        public HostInfo(string Ip, string Hostname, string Version) {
            this.Ip = Ip;
            this.Hostname = Hostname;
            this.Version = Version;
        }
    }
}

File: java-k8s-demo/src/main/java/com/k8s/demo/javak8sdemo/resource/HostInfo.java
package com.k8s.demo.javak8sdemo.resource;

public class HostInfo {
    
    private final String version;
    private final String ip;
    private final String hostname;

    public HostInfo(String version, String ip, String hostname) {
        this.version = version;
        this.ip = ip;
        this.hostname = hostname;
    }

    public String getIp() {
        return ip;
    }

    public String getHostname() {
        return hostname;
    }

    public String getVersion() {
        return version;
    }

}